### DISCUSSION TITLE
Release Publish & Tag Creation Failure

### DISCUSSION THREAD

**Discussion by CodeConfidant** [community/community#40535] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

When trying to create a new release/tag, error is displayed despite release/tag name being perfectly valid.

See Screenshot:

![image](https://user-images.githubusercontent.com/57354235/205027874-05043f8e-c00a-4227-a8d5-18bab516d51b.png)

**Comment by jfullstackdev** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/47092464/205066830-41951a84-9af8-401a-99c9-c0b6c3bc68ee.png)

![image](https://user-images.githubusercontent.com/47092464/205066901-f61eb35f-3f49-4485-bafc-d195b6128a8e.png)

![image](https://user-images.githubusercontent.com/47092464/205067002-4113f460-6498-4e71-b407-fa6de2649202.png)

you need to click Choose Tag, either from an existing or create a new one, I got the same error by leaving it blank

**Reply by CodeConfidant** [reply]
Got it, I see why it wasn't working. I thought the intended functionality was to infer & create a tag from the release title rather than having to create it via drop-down.
